metriq_gym/run.py

```diff
@@ -19,7 +17,15 @@ def main():
     params = load_and_validate(args.input_file)
     job_manager = JobManager(args.jobs_file)
 
-    HANDLERS[JobType(params["benchmark_name"])](args, params, job_manager).dispatch_handler()
+    handler = HANDLERS[JobType(params["benchmark_name"])](args, params, job_manager)
```

Comment:
The main problem I see with this PR is that in the case of polling we don't want to pass an input_file specifying the paramenter of the job, but we want to pass the job_id that was created at dispatch time. 
```
python run.py poll <job_id>
```
The mechanism I envision is that, given the `job_id`, the `JobManager` should fetch the correct  benchmark_name.

Reply:
This is a good point, and it has been adapted such that polling does not require an input_file (as this doesn't make sense for the polling action).